cornflow/models/meta_model.py

```diff
@@ -76,6 +81,23 @@ def update(self, data):
             setattr(self, key, item)
         super().update(data)
 
+    def patch(self, data):
+        """
+
+        :param dict data:
+        :type data:
+        """
+        try:
+            self.data = jsonpatch.apply_patch(self.data, data.get("patch"))
+        except jsonpatch.JsonPatchConflict as e:
+            raise InvalidPatch(payload={"message": str(e)})
```

Comment:
I think we usually pass the error message in the error attribute of the error and not in the payload. We don't use the payload except for AirflowErrors (not sure why). Check `InvalidUsage(error=val_err.normalized_messages())` in `endpoints/instance.py`. You can do: `InvalidPatch(error=str(e))` or `InvalidPatch(error=e.normalized_messages())` if that is available.

Reply:
Ok, the error message is quite huge but i did not try with the normalized_message. Will try it out and pass it as the error, not the message.

Reply:
Ok, so the exception raised by jsonpatch does not have an implementation of normalized message, so I am going to change it so it does not give back the full error in the messages attribute and we can think later if we want to capture the error and how.